lib/collection/src/shards/local_shard/mod.rs

```diff
@@ -934,85 +932,105 @@ impl Drop for LocalShard {
 /// Convenience struct for combining clock maps belonging to a shard
 ///
 /// Holds a clock map for tracking the highest clocks and the cutoff clocks.
-#[derive(Clone)]
+#[derive(Clone, Debug, Default)]
 pub struct LocalShardClocks {
-    highest_clocks: Arc<Mutex<ClockMap>>,
-    cutoff_clocks: Arc<Mutex<ClockMap>>,
-    highest_clocks_path: PathBuf,
-    cutoff_clocks_path: PathBuf,
+    newest_observed_clocks: Arc<Mutex<ClockMap>>,
+    oldest_resolvable_clocks: Arc<Mutex<ClockMap>>,
```

Comment:
I like short ones better, maybe even: `newest`, `oldest`

We can explain details in comments if we like.